website/notifications/emails.py

```diff
@@ -124,6 +126,11 @@ def check_node(node, event):
     return node_subscriptions
 
 
+def check_target_user(user, event):
```

Comment:
Yes, it's getting all of the user's subscriptions for that particular event type (but not every event they are subscribed to).